unlock-app/src/components/interface/checkout/main/Select.tsx

```diff
@@ -368,7 +376,8 @@ export function Select({ checkoutService, injectedProvider }: Props) {
                     lock,
                     existingMember: !!membership?.member,
                     skipQuantity,
-                    // unlock account are unable to renew
+                    skipRecipient,
+                    // unlock account are unable to renew : wut?
```

Comment:
We don't have an endpoint for this because we would need to trigger the transaction on their behalf. 

We decided to discourage unlock account users to pay using crypto or fund their wallet created by us previously.

Reply:
> We don't have an endpoint for this because we would need to trigger the transaction on their behalf.

I am confused. On the API call that calls `purchase`, we could easily check if the recipient already has an expired membership NFT and use `extendMembershipFor` if they do?

Reply:
We could. We didn't add it at the time and disabled unlock accounts from using crypto version.

Reply:
Ok I'll try to work on this in a different PR!